source/mongodb-handshake/handshake.md

```diff
@@ -405,6 +405,9 @@ class DriverInfoOptions {
 }
 ```
 
+Two `DriverInfoOptions` objects are considered identical if all fields are strictly equal with case sensitive string
```

Comment:
It just occurred to me that we may want to adjust this wording further to account for the null cases: perhaps, "if all non-empty fields". But the testing will likely want to be deferred to DRIVERS-3251 - thoughts?

Reply:
From async conversation, expanded wording to: "Two DriverInfoOptions objects are considered equal if they would result in the same metadata in the client handshake.  In practice, this means if a field is the empty string (""), treat it as unset.  Assert that each field is strictly equal with case sensitive string comparison." Planning to add test cases, but only assert on the resulting metadata for the non-empty fields.

Reply:
Done